packages/enzyme/src/ShallowWrapper.js

```diff
@@ -1683,7 +1698,7 @@ class ShallowWrapper {
    * @param {Object} options
    * @returns {ShallowWrapper}
    */
-  dive(options = {}) {
+  dive(options) {
```

Comment:
the options are optional, so the default value is correct (this change would change the length of the function)
```suggestion
  dive(options = {}) {
```